src/sgame/sg_spawn_mover.cpp

```diff
@@ -598,19 +604,28 @@ bool IsAutomaticMover( const gentity_t *ent )
 // that is doors that are not automatic, or doors that are already opened.
 void BotHandleDoor( gentity_t *ent, moverState_t moverState )
 {
+	if ( !IsDoor( ent ) || IsAutomaticMover( ent ) )
+	{
+		return;
+	}
+	glm::vec3 mins = VEC2GLM( ent->r.absmin );
+	glm::vec3 maxs = VEC2GLM( ent->r.absmax );
+	// this is a hack that allows to ignore "irrelevant" doors,
+	// implemented for nova's electric fence support
 	switch ( moverState )
 	{
 		case MOVER_POS1:
 		case MOVER_POS2:
 		case ROTATOR_POS1:
 		case ROTATOR_POS2:
-			if ( IsDoor( ent ) )
+			if ( false
```

Comment:
what's the if false for?

Reply:
Oh, it's nothing in itself, just a trick of mine.
Basically, using false on the if's line, and putting all OR'd condition (or true for AND'd) on their own line allows text editors to automatically align the code (I only do that for long lines).
It also easily allows to comment one part of the conditions for debug purposes.
Forgot to remove it before pushing to this repo.